src/commonComponents/BreadCrumb/BreadCrumb.js

```diff
@@ -0,0 +1,30 @@
+import "./BreadCrumb.css";
+import Link from "./components/Link";
+import PropTypes from "prop-types";
+
+const renderLinks = function (items) {
+  return items.map((item) => {
+    return <Link key={item.name} item={item} />;
```

Comment:
Although this is fine for most of the cases, we should not rely on name to be unique always